trunk/SUAVE/Methods/Weights/Correlations/General_Aviation/fuselage.py

```diff
@@ -0,0 +1,62 @@
+## @ingroup Methods-Weights-Correlations-General_Aviation
+# fuselage.py
+#
+# Created:  Mar 2016, M. Vegh
+# Modified: 
+
+# ----------------------------------------------------------------------
+#  Imports
+# ----------------------------------------------------------------------
+
+from SUAVE.Core import Units
+import numpy as np
+
+# ----------------------------------------------------------------------
+#   Fuselage
+# ----------------------------------------------------------------------
+## @ingroup Methods-Weights-Correlations-General_Aviation
+def fuselage(S_fus, Nult, TOW, w_fus, h_fus, l_fus, l_ht, q_c, V_fuse, diff_p_fus):
+    """ 
+        Calculate the weight of a fuselage for a GA aircraft
```

Comment:
Add a source, see above comment for ordering of comments.

Reply:
done

Reply:
I think when possible we should start adding sources by page etc. These weight correlations are on page 590 in the fifth edition.

Reply:
Done. I used the 4th edition though